kvdb-rocksdb/src/lib.rs

```diff
@@ -166,6 +166,13 @@ pub struct DatabaseConfig {
 	/// It can have a negative performance impact up to 10% according to
 	/// https://github.com/facebook/rocksdb/wiki/Statistics.
 	pub enable_statistics: bool,
+	/// Open the database in secondary mode
```

Comment:
should be nice to mention that secondary instance is read-only and provide the link to wiki https://github.com/facebook/rocksdb/wiki/Secondary-instance